tests/sync_tests/requests_test.py

```diff
@@ -0,0 +1,172 @@
+import httpx
+import pytest
+from unittest.mock import MagicMock, patch
+
+from ctfdpy.ctfdpy.http import HTTPClient, HTTPMethod
+from ctfdpy.ctfdpy.exceptions import HTTPError
+
+@pytest.mark.anyio
+@patch("ctfdpy.ctfdpy.http.HTTPClient._call")
+def test_request_call(mocker: MagicMock):
```

Comment:
You had added a decorator to signal the use of asynchronism, but these decorated functions are synchronous. Is is normal?

Reply:
Bad understanding of what it was. Removing the markers